shared-libs/validation/src/validation.js

```diff
@@ -178,6 +178,30 @@ module.exports = {
       logger.error('isISOWeek validation failed: the number of week is greater than the maximum');
       return Promise.resolve(false);
     },
+
+    isAfter: (doc, validation) => {
+      const fields = [...validation.funcArgs];
+      const duration = _parseDuration(fields.pop());
+      const testDate = moment(doc[validation.field]);
```

Comment:
It's possible that the field does not contain a valid date. I think we should error in this case.

Reply:
Added checks.